src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -756,12 +770,42 @@ export function generateTooltipFn(
             textData[0][0] = customEscape(point.name);
         }
 
-        return `<table class="tt-values gd-viz-tooltip-table">${textData.map(line => (
-            `<tr class="gd-viz-tooltip-table-row">
-                <td class="gd-viz-tooltip-table-cell title gd-viz-tooltip-table-title">${line[0]}</td>
-                <td class="gd-viz-tooltip-table-cell value gd-viz-tooltip-table-value">${line[1]}</td>
-            </tr>`
-        )).join('\n')}</table>`;
+        return renderTooltipHTML(textData);
+    };
+}
+
+export function generateTooltipTwoAttributesFn(
```

Comment:
it should be `generateTooltipForTwoAttributesFn`

Reply:
updated